packages/components/src/public/QueryModel/SelectionStatus.tsx

```diff
@@ -43,7 +43,7 @@ export const SelectionStatus: FC<RequiresModelAndActions> = memo(({ actions, mod
         selectionCount = (
             <span className="selection-status__count">
                 {selectionSize.toLocaleString()} of{' '}
-                {isLoadingTotalCount ? <LoadingSpinner msg="" /> : rowCount.toLocaleString()} selected
+                {isLoadingTotalCount ? <LoadingSpinner msg="" /> : rowCount?.toLocaleString()} selected
```

Comment:
Not a big deal, but if `rowCount` is undefined, this is just going to show something like `3 of selected`. Perhaps
```suggestion
                {(isLoadingTotalCount || rowCount == null) ? <LoadingSpinner msg="" /> : rowCount.toLocaleString()} selected
```
Of, if we think `rowCount` could remain undefined, we could modify this to display something like `3 of ? selected`. But it's an edge case to be sure.

Reply:
Yeah, that would make sense as well. I don't have a repro for this scenario, so I'll likely just leave this as-is for now and we can adjust later as needed.